x/evm/keeper/msg_server.go

```diff
@@ -153,9 +163,64 @@ func (k *Keeper) UpdateParams(goCtx context.Context, req *types.MsgUpdateParams)
 	}
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	if err := k.updatePrecompiles(ctx, req.Params.EnabledPrecompiles); err != nil {
+		return nil, err
+	}
+
 	if err := k.SetParams(ctx, req.Params); err != nil {
 		return nil, err
 	}
```

Comment:
`updatePrecompiles()` naming a bit vague in usage here since it's also set in state in SetParams() - could optionally rename to something more specific / different from just updating/setting like initializePreocmpiles